db/machclass.c

```diff
@@ -55,6 +57,7 @@ int mach_class_init(void)
     Pthread_mutex_lock(&mach_mtx);
     classes = hash_init_strcaseptr(offsetof(struct machine_class, name));
     class_names = hash_init_i4(offsetof(struct machine_class, value));
+    class_names = hash_init_i4(offsetof(struct machine_class, value));
```

Comment:
Initialized twice?

Reply:
ugh .. corrected. Thanks for catching this!